### PR TITLE
Arohasina's pull request

### PR DESCRIPTION
**Pull request by Arohasina**
this is my complete code that draws, using the turtle module, a house with a roof and a window with a girl standing next to it and the sun shining.

### CODE REVIEW

File: hw03_arohasina.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    t.begin_fill()
+    for i in range (2):
+        t.forward(140)
+        t.left(90)
+        t.forward(210)
+        t.left(90)
+    t.end_fill()
+
+
+def window(wn,t):
```
Comment (sheggen) [inline hw03_arohasina.py:38]: function names are typically verbs, e.g., `draw_window()`

**Comment by Arohasina** [comment]
I changed the function names to verbs

**Comment by sheggen** [comment]
Code looks great! Let's move some files:

1. Move the **arohasina_hw03_questions.md** file into the _Answers_ folder. 
2. You made a tiny edit to our hw03_questions.md file that can't get pushed. Please revert that file. If you need help, come see me. 
3. Add your Google Doc to the _PDFs_ folder.

**Comment by Arohasina** [comment]
I moved Arohasina_hw03_questions_copy.md to the Answer folder, added my pdf to the pdfs folder and I think I reverted the hw03_questions.md file !

**Pull request merged**